client/landing/stepper/utils/get-flow-from-url.ts

```diff
@@ -1,16 +1,24 @@
-import { matchPath } from 'react-router-dom';
+export const getFlowFromURL = (
+	url = window.location.pathname,
```

Comment:
I do agree with adding the ability to pass in an optional pathname though, so that this utility function can be used on the server.

Maybe rename `url` to `pathname` so it's clear this should just be the path fragment?